### PR TITLE
100uptime

### PR DESCRIPTION
**Pull request by TiGrib**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+vk_api_token = '05050276e5d36e9c15140177eb77ef423f7fae4e5747ee764992a988a1371658aaff039e8baa32a7de585'
```
Comment (eamag) [inline config.py]: You shouldn't put credentials into git, use enviroment variables and keep it in some password manager

File: delkeyboard.json [hunk 2]
```diff
@@ -1 +1,4 @@
-{"buttons":[],"one_time":true}
\ No newline at end of file
+{
+  "buttons": [],
+  "one_time": true
+}
```
Comment (eamag) [inline delkeyboard.json]: Newline at the end of the file

File: server.py [hunk 3]
```diff
@@ -2,58 +2,43 @@
...
-        
-        self.vk=vk_api.VkApi(token=api_token)
-        #For using Long Poll API
-        self.long_poll=VkBotLongPoll(self.vk, group_id)
-        #for using vk_api methods
-        self.vk_api=self.vk.get_api()
-        
+
+        self.vk = vk_api.VkApi(token=api_token)
+        # For using Long Poll API
```
Comment (eamag) [inline server.py]: Useless comments, should be obvious from var names

File: server.py [hunk 4]
```diff
@@ -2,58 +2,43 @@
...
+
     def send_msg(self, send_id, message):
-        
-        return self.vk_api.messages.send(peer_id=send_id,message=message,keyboard=open('keyboard.json', 'r',encoding='UTF-8').read(), random_id=random.randint(0, 100000000))
-    
-    
+
+        return self.vk_api.messages.send(peer_id=send_id, message=message.lower(),
+                                         keyboard=open('keyboard.json', 'r', encoding='UTF-8').read(),
+                                         random_id=random.randint(0, 100000000))
```
Comment (eamag) [inline server.py]: random.randint(0, 100000000)
what's this for?

File: server.py [hunk 5]
```diff
@@ -2,58 +2,43 @@
...
-        
+
     def send_msg(self, send_id, message):
-        
-        return self.vk_api.messages.send(peer_id=send_id,message=message,keyboard=open('keyboard.json', 'r',encoding='UTF-8').read(), random_id=random.randint(0, 100000000))
-    
-    
+
+        return self.vk_api.messages.send(peer_id=send_id, message=message.lower(),
+                                         keyboard=open('keyboard.json', 'r', encoding='UTF-8').read(),
```
Comment (eamag) [inline server.py]: context manager with open

File: server_manager.py [hunk 6]
```diff
@@ -2,23 +2,9 @@
...
+from server import Server
 
-
-
-
-
-server1=Server(vk_api_token, 176452593, 'server1')
+server1 = Server(vk_api_token, 176452593, 'server1')
 
 server1.start()
```
Comment (eamag) [inline server_manager.py]: google: if name main

File: server.py (hunk 4 above)
Comment (TiGrib) [reply server.py]: я уже объяснял что в вк к каждому сообщению обязательный параметр random_id

File: delkeyboard.json (hunk 2 above)
Comment (TiGrib) [reply delkeyboard.json]: pycharm через автоформатирование текста так сделал

File: server.py (hunk 4 above)
Comment (eamag) [reply server.py]: What if you get same id for two different messages?

File: server.py [hunk 7]
```diff
@@ -2,58 +2,46 @@
...
 
-from vk_api.bot_longpoll import VkBotLongPoll
+import vk_api.vk_api
 from vk_api.bot_longpoll import VkBotEventType
+from vk_api.bot_longpoll import VkBotLongPoll
 
 from commander import Commander
 
+with open('keyboard.json', 'r', encoding='UTF-8') as keyboard_file:
+    keyboard = keyboard_file.read()
```
Comment (eamag) [inline server.py]: should be in the class, not global. Put it into send_msg

File: server.py [hunk 8]
```diff
@@ -2,58 +2,46 @@
...
-        
-        
-
-
-
-
-
 
+                if event.type == VkBotEventType.MESSAGE_NEW:
+                    self.send_msg(event.object.from_id, self.users[event.object.from_id].input(event.object.text))
```
Comment (eamag) [inline server.py]: Why do you need to check it twice?

Comment (TiGrib) [reply server.py]: first time it's for new users who start conversation

File: server.py (hunk 7 above)
Comment (TiGrib) [reply server.py]: done

File: server.py (hunk 8 above)
Comment (eamag) [reply server.py]: You have 
`            if event.type == VkBotEventType.MESSAGE_NEW:`
twice in your code`

**Pull request merged**